src/components/BucketsList.tsx

```diff
@@ -32,7 +32,11 @@ export const BucketsList = (): ReactElement => {
           {buckets && <span className="font-normal">({buckets.length})</span>}
         </PageTitle>
       </div>
-      {!loading && error && <pre>error: {JSON.stringify(error, null, 2)}</pre>}
+      {!loading && error && (
+        <pre className="mb-4">
+          error: {JSON.stringify(error.message, null, 2)}
```

Comment:
I know we are still testing, but if we are using the message of the error, we can just display it, and not the stratified version of it.
```
{!loading && error && <div>{error.message}</div>}
```